packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts

```diff
@@ -0,0 +1,17 @@
+import '@aws-cdk/assert/jest';
+import * as elbv2 from '../../lib';
+
+describe('tests', () => {
+  test('pathPatterns length should be less than or equal to 5', () => {
```

Comment:
Can you add a positive case as well?

Reply:
I see a commit labeled "add positive case", but don't see it in the diff yet. Did it get lost in a merge somewhere?

Reply:
The unit test has been updated, please check whether it meets the requirements, thanks